Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeTokenizationClient.swift

```diff
@@ -0,0 +1,36 @@
+import BraintreeCore
+import PayPalCheckout
+import BraintreePayPal
+
+class BTPayPalNativeTokenizationClient {
+
+    private let apiClient: BTAPIClient
+
+    init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    func tokenize(returnURL: String,
+                  request: BTPayPalRequest,
+                  completion: @escaping (Result<BTPayPalNativeAccountNonce, BTPayPalNativeError>) -> Void) {
+
+        let tokenizationRequest = BTPayPalNativeTokenizationRequest(returnURL: returnURL,
+                                                                    request: request,
+                                                                    correlationID: "", // TODO: get this value from PayPalCheckout SDK
+                                                                    clientMetadata: apiClient.metadata)
+
+        apiClient .post("/v1/payment_methods/paypal_accounts", parameters: tokenizationRequest.parameters()) { body, _, error in
```

Comment:
```suggestion
        apiClient.post("/v1/payment_methods/paypal_accounts", parameters: tokenizationRequest.parameters()) { body, _, error in
```

Reply:
I'm unresolving this because it doesn't look like the change was actually committed?